src/host/getset.cpp

```diff
@@ -835,18 +839,20 @@ void ApiRoutines::GetLargestConsoleWindowSizeImpl(const SCREEN_INFORMATION& cont
         //      this scenario is highly unlikely, and we can reasonably do this
         //      on their behalf.
         // see MSFT:19853701
+
+        // use context (not buffer) here because alternate buffer may not be aware of VT mode

```

Comment:
So chatting with @zadjii-msft made me more confident that it's fine to use `buffer` here. It's even _more_ possible that this code won't even be necessary with ScrollRegion being as okay as it is

Reply:
Oh gods we investigated this
It's true, it is correct, the alt buffer has the same modes as the main buffer. It is propagated incidentally through a magic global. :smile:

Reply:
Incidentally and in a way that could make it look like an accident.